web/blog/parquet-and-feather-enabling-open-investigations/index.md

```diff
@@ -4,6 +4,7 @@ authors:
   - mavam
   - dispanser
 date: 2022-10-07
+last_updated: 2022-10-22
```

Comment:
The generated `index.md` file is not ignored by Git, can you add a `.gitignore` file that just contains `/index.md` into this directory?

Reply:
Note that this is the *previous* blog post that is not compiled from a `qmd` file.

I guess it's good and bad that it's in theory indistinguishable from a Quarto notebook. 😅

Reply:
Ah damnit, I just left the comment on the wrong file. I meant to add it to the current blog post.

Reply:
I'll transpose this comment accordingly.